udata/tests/api/test_auth_api.py

```diff
@@ -582,7 +584,7 @@ def test_invalid_implicit_grant_token(self, client, oauth):
         )
 
         assert_status(response, 400)
-        assert response.json["error"] == "invalid_grant"
+        assert response.json["error"] == "unsupported_response_type"
```

Comment:
Was this error modification expected?

Reply:
From what i can see from the source code, the error returned has been changed between [0.14.3](https://github.com/lepture/authlib/blob/v0.14.3/authlib/oauth2/rfc6749/authorization_server.py#L142-L151)  and [1.3.1](https://github.com/lepture/authlib/blob/v1.3.1/authlib/oauth2/rfc6749/authorization_server.py#L198-L207) in the same `get_authorization_grant` function.